pages/linux/audit2allow.md

```diff
@@ -0,0 +1,16 @@
+# audit2allow
+
+> Create an SELinux local policy module to allow rules based on denied operations found in logs.
+> More information: <https://man7.org/linux/man-pages/man1/audit2why.1.html>.
```

Comment:
```suggestion
> More information: <https://manned.org/audit2allow>.
```
We use manned for man pages, see style guide

Reply:
Noted!